packages/terra-form-fieldset/src/Fieldset.scss

```diff
@@ -0,0 +1,57 @@
+:local {
+  .field,
+  .fieldset {
+    border: 0;
+    margin-bottom: 1.428em;
+    padding: 0;
+  }
+
+  @media screen and (min-width: 544px) {
+    .field-inline,
+    .fieldset-inline {
+      display: inline-block;
+      margin-right: 0.7143em;
+      vertical-align: top;
+    }
+  }
+
+  .field-required > .label:first-child,
+  .fieldset-required > .legend:first-child {
+    &::before {
+      color: #e50000;
```

Comment:
We should make this color themeable. Not all locales and themes want the required text to be red.